tests/managed/methods.cs

```diff
@@ -88,6 +88,12 @@ public void Remove (Item item)
 		}
 	}
 
+	public class DuplicateMethods {
+		public int DoIt () { return 42; }
+		public int DoIt (int i) { return 42; }
+		public int DoIt (string s) { return 42; }	
```

Comment:
oh that's 99% great :) I just assumed this would be shared between .ctor/method.

This
`- (int)doItInt32Int32:(int) i;`
would be nicer as
`- (int)doItInt32:(int) i;`